src/ContactsDesktop/Controller.java

```diff
@@ -0,0 +1,58 @@
+package ContactsDesktop;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Alert;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TextField;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class Controller implements Initializable{
+    ObservableList<Contact> contacts = FXCollections.observableArrayList();
+
+    @FXML
+    ListView list;
+
+    @FXML
+    TextField name;
+
+    @FXML
+    TextField phone;
+
+    @FXML
+    TextField email;
+
+    public void addContact(){
+        String contactName = name.getText();
+        String contactPhone = phone.getText();
+        String contactEmail = email.getText();
+        Contact contact = new Contact(contactName, contactPhone, contactEmail);
+        if((!contactName.isEmpty() && !contactPhone.isEmpty() && !contactEmail.isEmpty())) {
```

Comment:
This test should probably be put into the `Contact` class as an `isValid` class.  The method would centralize the validation logic across the application so if more validation is needed it only requires code changes in one spot.